modules/claims_api/app/controllers/claims_api/v1/forms/power_of_attorney_controller.rb

```diff
@@ -237,16 +237,11 @@ def build_representative_info(poa_code)
 
         def select_service(ssn)
           # rubocop:disable Rails/DynamicFindBy
-          if Flipper.enabled? :bgs_via_faraday_file_number
-            ClaimsApi::Logger.log('poa', detail: 'local BGS service used to locate file_number')
-            ClaimsApi::LocalBGS.new(
-              external_uid: target_veteran.participant_id,
-              external_key: target_veteran.participant_id
-            ).find_by_ssn(ssn)
-          else
-            ClaimsApi::Logger.log('poa', detail: 'bgs-ext used to locate file_number')
-            bgs_service.people.find_by_ssn(ssn)
-          end
+          ClaimsApi::Logger.log('poa', detail: 'local BGS service used to locate file_number')
+          ClaimsApi::LocalBGS.new(
+            external_uid: target_veteran.participant_id,
+            external_key: target_veteran.participant_id
+          ).find_by_ssn(ssn)
           # rubocop:enable Rails/DynamicFindBy
         end
```

Comment:
I would say we remove select service and the logging as we know we're using local_bgs now and just pass this straight to the call in `check_file_number_exists!`